README.md

```diff
@@ -1 +1,67 @@
-# my-portfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Dyary's Portfolio
+
+> this is my portfolio website. It includes info about me, how to contact me and my achivemets and creations as a web developer.
+
+![screenshot](https://i.imgur.com/vWXwTaR.png)
+
+look thorugh the website to see my work and contact me if need.
+
+## Built With
+
+- HTML and CSS
+- VS Code, Chrome DevTools
+
+## Live Demo
+
+[Live Demo Link](https://i.imgur.com/vWXwTaR.png)
```

Comment:
If you don't have a link where your web app is deployed, you can remove the Live Demo section from your README file, or you can use Github Pages to deploy your app for free.

- [ ] Remove the Live Demo section from the README file or add the right link where your web app is deployed.